box-ios-sdk.podspec

```diff
@@ -20,13 +20,13 @@ s.ios.source_files        = "BoxContentSDK/BoxContentSDK/*.{h,m}", "BoxContentSD
 s.ios.exclude_files       = "BoxContentSDK/BoxContentSDK/External/ISO8601DateFormatter/BOXISO8601DateFormatter.{h,m}",
 "BoxContentSDK/BoxContentSDK/External/KeychainItemWrapper/BOXKeychainItemWrapper.{h,m}"
 s.ios.public_header_files = "BoxContentSDK/BoxContentSDK/*.h", "BoxContentSDK/BoxContentSDK/**/*.h"
-s.resource_bundle = {
```

Comment:
we don't have any resources to bundle with content sdk, but having bundle resources without Info.plist will lead to the codesign error we saw when building the project.